### DISCUSSION TITLE
turn to symfony/mailer

### DISCUSSION THREAD

**Discussion by jyxjjj** [laravel/framework#38572] [Q&A] [answered]
Swift Mailer will stop maintain.
They suggested turn to symfony/mailer.
Did laravel already has this target?

**Comment by aneeskhan47** [comment, marked as answer]
@jyxjjj I think they started to work on it #38481

**Comment by jyxjjj** [comment]
Hmmm. There's an error occurred this github webpage. I cannot mark as answer. (HTTP 422)
Or maybe just i have no permission.
But thank you!
